server/eclipse-project/src/main/java/us/freeandfair/corla/model/CountyDashboard.java

```diff
@@ -484,24 +548,28 @@ public void startRound(final int the_number_of_ballots,
                                   the_audit_subsequence);
     my_rounds.add(round);
   }
-  
+
   /**
    * Ends the current round.
-   * 
-   * @param the_signatories The signatories for round sign-off.
+   *
+   * Signs out all audit boards, and performs any bookkeeping necessary to end
+   * the round.
+   *
    * @exception IllegalStateException if there is no current round.
    */
-  public void endRound(final List<Elector> the_signatories) {
+  public void endRound() {
     if (my_current_round_index == null) {
       throw new IllegalStateException("no round to end");
     } else {
+      this.setAuditBoardCount(null);
```

Comment:
Will the County Admin have to set the AuditBoardCount on the next round? Why not keep it around?

Reply:
The thought was that the county might be able to send audit boards home.

Reply:
We need to test round transitions a bit better, maybe we can revisit this, but the actual setting of the audit board count is what triggers the code to map ballots onto boards as well.